corehq/apps/export/models/new.py

```diff
@@ -608,6 +617,14 @@ def _merge_schemas(cls, *schemas):
         schema = cls()
 
         def resolvefn(group_schema1, group_schema2):
+
+            def keyfn(export_item):
```

Comment:
might be good to add a test case that properly resolves two items with the same path and doc_type but different transforms